Formula/joplin.rb

```diff
@@ -0,0 +1,20 @@
+require "language/node"
+
+class Joplin < Formula
+  desc "Joplin, a free, open source note taking and to-do application"
```

Comment:
We know it's open-source otherwise it wouldn't be in Homebrew. Also the description should not start with the formula's name.